src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
-import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard
+          title={movie.title}
```

Comment:
Here you can use destructuring, but this is of course optional. Overall the work is excellent!